ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -119,10 +124,17 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
     private leoNotebooksService: NotebooksService,
     private jupyterService: JupyterService,
     private sanitizer: DomSanitizer
-  ) {}
+  ) {
+    const wsData: WorkspaceData = this.route.snapshot.data.workspace;
+    this.accessLevel = wsData.accessLevel;
```

Comment:
I realize this is still WIP, but would save us work down the road if this were instead a queryParam per the ticket description. Eventually we want to support launching **any** notebook in playground mode (as an option; it would only be the default behavior for read-only notebooks) , so ideally the playground mode decision should not happen here.